packages/manager/core/generator/app/templates/src/index.tsx.hbs

```diff
@@ -10,8 +10,19 @@ import '@ovhcloud/ods-theme-blue-jeans/dist/index.css';
 import './global.css';
 import './vite-hmr';
 
+import { UNIVERSE, SUB_UNIVERSE, APP_NAME, LEVEL2 } from './tracking.constant';
+
+const trackingContext = {
+  chapter1: UNIVERSE,
+  chapter2: SUB_UNIVERSE,
+  chapter3: APP_NAME,
+  appName: APP_NAME,
+  level2: LEVEL2,
+  pageTheme: UNIVERSE,
```

Comment:
It seems to be different "DedicatedServers" for vRack Services
While UNIVERSE is "Baremetal"

Reply:
How do you fill this attribute ? Its only based on the tracking file ?

Reply:
Yes it looks like a constant defined by the tracking team

Reply:
👍